research-software-directory/RSD-as-a-service
# 627 realistic test data

# More realistic fake data

Changes proposed in this pull request:

* Project starting and end dates can be so that a project is either finished, running or starting
* Add more real DOI's for mentions
* Add real concept DOI's for software
* Add real ROR id's for organisation
* On all places where images are used, we use local images in the `img` directory, instead of fetching images from the internet.


How to test:
* `docker-compose down --volumes && docker-compose build --parallel && docker-compose up --scale data-generation=1`
* Visit various pages on http://localhost
	* Look at the images on software pages
	* Look at the organisations overview, inspect the images
	* Visit some organisation pages and look at the ROR info
	* Look at the project overview, see the different statuses
* Check that the releases scraper (`docker-compose exec scrapers python3 -u /usr/myjava/releases.py`) doesn't just complain that the conceptDOI could not be found


Closes #627

PR Checklist:
